docarray/array/mixins/embed.py

```diff
@@ -93,8 +126,15 @@ def _set_embeddings_paddle(
         embed_model.to(device=device)
         embed_model.eval()
         for b_ids in self.batch_ids(batch_size):
-            batch_inputs = paddle.to_tensor(self[b_ids, 'tensor'], place=device)
-            r = embed_model(batch_inputs)
+            batch_inputs = collate_fn(self[b_ids])
+            if isinstance(batch_inputs, Mapping):
```

Comment:
this `batch_inputs` in Pytorch and Tensorflow are hugging face `transformer` `BatchEncoding` and `BatchFeature` classes, see [here](https://github.com/jina-ai/finetuner.fit/blob/main/finetuner/device.py#L21), I'm not sure how paddle resolve this.

Reply:
same in paddle, this is not only about `transformers`. this is implemented for kwargs parameters in `forward` method

Reply:
and of course, it can't work at all cases. For example, the current implementation cannot work with `sentence_transformers` since its forward method accepts *dict* as the whole input.